### DISCUSSION TITLE
Release 5.2.0: Gpg key issues DP provisioning jobs

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#545] [Bugs] [answered]
Hi Team!
@santhosh-tg 
While provisiong below jobs are given gpg errors.
(Provision/dev/DataPipeline/KafkaIndexer, Provision/dev/DataPipeline/InfluxDB)
####
Failed to update apt cache: W:Updating from such a repository can't be done securely, and is therefore disabled by default., W:See apt-secure(8) manpage for repository creation and user configuration details., W:GPG error: https://repos.influxdata.com/ubuntu trusty InRelease.
#############

Solution:

sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D8FF8E1F7DF8B07E

**Comment by chitranshu-keshav** [comment]
@santhosh-tg can you please get the required documentation updated and get this thread closed

**Reply by santhosh-tg** [reply, marked as answer]
Hi @gohilamariappan a better solution would be to update it with new signing keys. Please refer https://www.influxdata.com/blog/linux-package-signing-key-rotation/  . It would be great if you can raise a PR for the same.

**Reply by Krishnaj20** [reply]
@gandham-santhosh - what is the status of this thread? If resolved, pls mark it as answered.

**Reply by gandham-santhosh** [reply]
@Krishnaj20 this is resolved so marking it as answered
